javascript-practice/src/styles/base/typography.css

```diff
@@ -0,0 +1,13 @@
+@font-face {
+  font-family: 'inter';
+  src: url('../../assets/fonts/Inter/Inter-Regular.woff2') format('woff2');
```

Comment:
Should add more fonts such as `.ttf` so it can support multiple browsers instead of only using a font `.woff2`